wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -412,11 +419,11 @@ def run(
 
 
 def maybe_create_imagepull_secret(
-    core_api: "CoreV1Api",
+    core_api: CoreV1Api,  # type: ignore
```

Comment:
Do we still need these type ignores?

Reply:
Yeah, it still complains about CoreV1Api etc not being a valid type. I also tried changing it to `core_api: Type[CoreV1Api]` and `core_api: "CoreV1Api"` but both give the same message.